src/core/store/reducers/globals.js

```diff
@@ -21,6 +22,8 @@ export default function globals(state = initialState, action) {
       return {...state, sms: action.payload}
     case constants.SET_CLIENT_SUCCESS:
       return {...state, clientSuccess: action.payload}
+    case constants.SET_I18N:
```

Comment:
what is stored in redux, the i18n instance? If so, we should remove, in the store we should just have serialisable data.

Reply:
Yes, that's an instance of `polyglot`. I will get rid of it from the redux store, I can just create a new instance of `polyglot` when I receive the config on cross device client.

Reply:
cool. how will you pass the instance, parent to child?